src/main/java/com/sda/controller/FormController.java

```diff
@@ -0,0 +1,22 @@
+package com.sda.controller;
+
+import com.sda.model.Customer;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@Controller
+public class FormController {
+
+    @GetMapping("/rentForm")
+    public String sendForm(Customer customer) {
+        return "rentForm";
+    }
+
+    @PostMapping("/rentForm")
+    public String handleCostumeForm(@ModelAttribute Customer customer) {
```

Comment:
brakuje obslugi obiektu Customer w obu metodach. Zakladam że planujecie to na później?

Reply:
szczerze to Customer był zrobiony tylko i wyłącznie dla formatki, natomiast nie jest ona póki co wysyłana na żaden fakeowy mail. Czy w takim razie, z racji tego, że są też inne fixy do zrobienia, możemy ograniczyć się do takiej obsługi Customera?

Reply:
Obsluga obiektu Customer bedzie rozwiazana po zmergowaniu Adminsite do Mastera.